packages/wallet-adapter-core/src/utils/helpers.ts

```diff
@@ -38,5 +38,8 @@ export function generalizedErrorMessage(error: any): string {
 // Serializable, so if each argument is of an instance of a class
 // the extends Serializable - we know these are BCS arguments
 export const areBCSArguments = (args: any): boolean => {
+  // `every` returns true if the array is empty, so
+  // first check the array length
+  if (args.length === 0) return false;
   return args.every((arg: any) => arg instanceof Serializable);
 };
```

Comment:
Is args an array of any?  We should probably fix the input argument to the function.